src/main.rs

```diff
@@ -114,12 +114,25 @@ fn main() {
 
             info!("Sent command: Run");
             replies.push(comm);
-            enclave_process_handle_all_replies::<EnclaveRunInfo>(&mut replies, 0, false, vec![0])
-                .map_err(|e| {
-                    e.add_subaction("Failed to handle all enclave process replies".to_string())
-                        .set_action(RUN_ENCLAVE_STR.to_string())
-                })
-                .ok_or_exit_with_errno(None);
+            let run_info = enclave_process_handle_all_replies::<EnclaveRunInfo>(
+                &mut replies,
+                0,
+                false,
+                vec![0],
+            )
+            .map_err(|e| {
+                e.add_subaction("Failed to handle all enclave process replies".to_string())
+                    .set_action(RUN_ENCLAVE_STR.to_string())
+            })
+            .ok_or_exit_with_errno(None);
+            if run_args.attach_console {
+                console_enclaves(run_info[0].enclave_cid, None)
```

Comment:
Nit: we are theoretically guaranteed at this point that `run_info[0]` exists and is a valid `EnclaveRunInfo` object. But I would also add a check for this, for good measure.